### PR TITLE
[DOCS] - Phoenix example

### PR DESCRIPTION
**Pull request by jnatherley**
Improve the documentation to show how it could work with Phoenix and some typos.

P.S Sweet lib, would love to see if there are ways i could contribute.

### CODE REVIEW

**Comment by keathley** [comment]
Hey @jnatherley, sorry for the delay. I've been distracted by tons of junk. I'll get this reviewed here soon. I glanced over it and it looked good but just wanna double check :). I'll put some issues in the list and some of those might be fine to work on if you're interested.

File: README.md [hunk 1]
```diff
@@ -206,6 +204,49 @@ defmodule AnotherService.GetHats do
...
 
+### Running with Phoenix
+
+The plug can also be attached within a Phoenix Router. Example below would be accessible at `/rpc/hat` 
+
+URL: `/rpc/hat/twirp/example.Haberdasher.MakeHat` or `/{prefix?}/twirp/{package}.{service}/{method}`
+
+```elixir
+defmodule ExampleWeb.Router do
+  use FlytroHealthWeb, :router
```
Comment (keathley) [inline README.md]: We can probably update this to ExampleWeb ;)

Comment (jnatherley) [reply README.md]: Done and thanks for the review @keathley

**Comment by keathley** [comment]
Thanks @jnatherley. This has been merged into master.

**Pull request closed without merging**